docs/06-concepts/18-testing/02-the-basics.md

```diff
@@ -228,17 +238,17 @@ In production, the transaction call will throw if any database exception happene
 
 The run mode that Serverpod should be running in. Defaults to `test`.
 
-### `enableSessionLogging`
+### `serverpodLoggingMode`
```

Comment:
Suggestion: Please modify the tables so that they match tables on other pages. Such as https://docs.serverpod.dev/next/concepts/authentication/setup#configure-authentication

Reply:
f9432a6